lib/pluginmanager/main.rb

```diff
@@ -32,6 +33,7 @@ class Main < Clamp::Command
       subcommand "pack", "Package currently installed plugins", LogStash::PluginManager::Pack
       subcommand "unpack", "Unpack packaged plugins", LogStash::PluginManager::Unpack
       subcommand "list", "List all installed plugins", LogStash::PluginManager::List
+      subcommand "new", "Creates a new plugins scaffold", LogStash::PluginManager::New
```

Comment:
"Create a new plugin structure". Scaffolding may only be familiar to ruby developers